src/tribler/core/components/bandwidth_accounting/tests/test_community.py

```diff
@@ -28,128 +30,121 @@ def create_node(self):
                         settings=BandwidthAccountingSettings())
         return ipv8
 
+    def database(self, i):
+        return self.overlay(i).database
+
+    def add_cache(self, i, cache):
+        return self.overlay(i).request_cache.add(cache)
+
     async def test_single_transaction(self):
         """
         Test a simple transaction between two parties.
         """
-        await self.nodes[0].overlay.do_payout(self.nodes[1].overlay.my_peer, 1024)
+        await self.overlay(ID1).do_payout(self.overlay(ID2).my_peer, 1024)
```

Comment:
```suggestion
        await self.overlay(ID1).do_payout(self.peer(ID2), 1024)
```

I forgot to change the `my_peer` to the "public" `self.peer()` here.